### PR TITLE
feat(chart): add further github config to chart

### PR DESCRIPTION
**Pull request by vdice**
TODO:
 - [x] update docs once https://github.com/Azure/brigade-github-app/pull/13 lands

### CODE REVIEW

**Comment by technosophos** [comment]
Oh, my bad... I guess it should be `appID` and `installationID` instead of snake case `app_id`.... sorry about that.

**Comment by vdice** [comment]
@technosophos actually, for `checkSuiteOnPR` (`CHECK_SUITE_ON_PR` in code), I see that the value is processed by `strconv.Atoi()` -- therefore, do we need to change the chart/deployment values from `"true"/"false"` to integers?

**Comment by technosophos** [comment]
Good catch! Yes, I suppose we do

**Comment by vdice** [comment]
@technosophos added logic to parse `CHECK_SUITE_ON_PR` as bool from env; PTAL

File: cmd/github-gateway/server.go [hunk 1]
```diff
@@ -66,6 +66,20 @@ func main() {
 		log.Printf("Forked PRs will be built for roles %s", strings.Join(allowedAuthors, " | "))
 	}
 
+	envOrBool := func(env string, defaultVal bool) bool {
```
Comment (technosophos) [inline cmd/github-gateway/server.go]: Nice! That's a lot cleaner than mine.

**Comment by vdice** [comment]
(Last amended commit was just to update docs to specify setting `CHECK_SUITE_ON_PR` as boolean rather than int)

**Pull request merged**